File: classexercise/python/bankaccount.py
amount = 0
remainder = 0
balance = 0
decision = 0


while(decision != -1):
	decision = int(input('Enter 1 to deposit or 2 to withdraw or 3 to see balance or -1 to quit: '))

	
	if(decision == 1):
		deposit = int(input('Enter amount to deposit: '))
		amount += deposit
	

	elif(decision == 2):
		withdraw = int(input('Enter amount to withdraw: '))	
		if(withdraw > amount):
			print('Invalid balance')
		else:
			amount -= withdraw
	elif(decision == 3):
		print(amount)
	else: 
		break

File: classexercise/java/BankAccount.java
import java.util.Scanner;
public class BankAccount{
public static void main(String []args){
	Scanner input = new Scanner(System.in);

	int totalAmount = 0;
	int totalWithdraw = 0;
	int balance = 0;
	

	System.out.print("Enter 1 to deposit or 2 to withdraw or 3 to check Balance or -1 to quit: ");
	int decision = input.nextInt();
	

	while(decision != -1){
	
	if(decision == 1){ 
	System.out.print("Enter amount to deposit: ");
	int amount = input.nextInt();
	totalAmount += amount;
	
	}else if(decision == 2){
	System.out.print("Enter amount to withdraw: ");
	int withdraw = input.nextInt();
	if(withdraw > totalAmount) System.out.println("Invalid number");
	else totalAmount -= withdraw;
	
	}else if(decision == 3){
	System.out.println("Your balance is " +totalAmount);
	}else break;
	
	System.out.print("Enter 1 to deposit or 2 to withdraw or 3 to check Balance or -1 to quit: ");
	decision = input.nextInt();

	}

	
	


}
}